test/functional/ios/webdriver_tests.py

```diff
@@ -24,9 +24,18 @@ class WebDriverTests(unittest.TestCase):
     def setUp(self):
         desired_caps = desired_capabilities.get_desired_capabilities('UICatalog.app.zip')
         self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
+        desired_caps = desired_capabilities.get_desired_capabilities('UICatalog.app.zip')
+        desired_caps['deviceName'] = 'iPhone Xs Max'
+        desired_caps['wdaLocalPort'] = '8102'
+        self.driver1 = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
```

Comment:
this is going to slow down all existing tests in that module. Can we move the init of the second driver into a separate test?

Reply:
Changed.